app/serializers/team_serializer.rb

```diff
@@ -22,4 +22,9 @@ class TeamSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :private
 
   has_many :folders, serializer: FolderMinimalSerializer
+
+  has_many :user_favourite_teams do |serializer|
+    serializer.object.user_favourite_teams.list.where(user: scope)
+    # scope.user_favourite_teams
```

Comment:
Comment needed?

Reply:
Well noticed. These two lines do the exact same, I'm actually not completely content with how I have to get the records here. If you got a better idea feel free to help out, was gonna discuss this with @psunix afterwards though. That's why I kept it for the moment

Reply:
interesting problem :) how to avoid n+1 queries here ?

I would rather create a computed boolean flag if it's favourite or not.

maybe we have to preload all favourite teams first and then set the boolean flag on the entries ?